configs/linstor/yum-repos.conf.tmpl

```diff
@@ -0,0 +1,6 @@
+[xcpng-linstor]
+name=xcpng - linstor
+baseurl=https://repo.vates.tech/xcp-ng/8/8.3/linstor/@@RPMARCH@@/
+gpgkey=https://xcp-ng.org/RPM-GPG-KEY-xcpng
+failovermethod=priority
+skip_if_unavailable=False
```

Comment:
That last one is the default documented value already, we can surely drop it (and from everywhere else).

OTOH it is missing `skip_missing_names_on_install=0`